vast-perf.sh

```diff
@@ -90,6 +90,7 @@ ADMINUSER="admin"
 ADMINPASSWORD=123456
 LOOPBACK=1 # only applies when running on cnodes. default is on now. BUT: this requires a lot of vips..
 VERBOSE=1 # verbosely output details, may be too noisy on large clusters
+file_name="randrw"
```

Comment:
This ends up being the prefix for the filename, but it's actually the fio job name. I'd prefer to see it be called `JOB_NAME` so users aren't surprised when this isn't actually the name of the file that gets created.